File: 태성/LV1/0604/소수만들기.py
def solution(nums):
    answer = 0

    pre = []
    for i in range(len(nums) - 2):
        # pre.append(nums[i])
        for j in range(i + 1, len(nums) - 1):
            # pre.append(nums[j])
            for k in range(j + 1, len(nums)):
                pre.append(nums[i] + nums[j] + nums[k])

    # print(pre)
    chk = []
    for p in pre:
        for i in range(2, (p // 2) + 1):
            if p%i == 0:
                chk.append(p)
                break
    answer = len(pre) - len(chk)
    return answer

File: 태성/LV1/0604/소수만들기.js
const nums = [1,2,3,4]

let pre = []

for (let i=0; i<(nums.length-2); i++) {
    for (let j=i+1; j<(nums.length-1); j++){
        for (let k=j+1; k<(nums.length); k++){
            // console.log(nums[i])
            // console.log(nums[j])
            // console.log(nums[k])
            pre.push(nums[i] + nums[j] + nums[k])
            // console.log(pre)
        }
    }
}
// console.log(pre)
let chk = []

for (let p of pre) {
    for (let i=2; i<(p/2)+1; i++){
        if (p%i === 0){
            chk.push(p)
            break
        }
    }
}
// console.log(pre.length)
// console.log(chk.length)

answer = pre.length - chk.length

console.log(answer)